<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphMainWindow</class>
 <widget class="QMainWindow" name="GraphMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveJSON"/>
    <addaction name="actionSaveCSV"/>
    <addaction name="actionSaveImage"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveJSON">
   <property name="text">
    <string>Сохранить JSON</string>
   </property>
  </action>
  <action name="actionSaveCSV">
   <property name="text">
    <string>Сохранить в CSV</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Сохранить изображение</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
